src/renderer/components/subscription-settings/subscription-settings.vue

```diff
@@ -2,25 +2,29 @@
   <ft-settings-section
     :title="$t('Settings.Subscription Settings.Subscription Settings')"
   >
-    <ft-flex-box class="settingsFlexStart500px">
```

Comment:
Can you please remove the ft-flex-box component and import from the `.js` file as well please.

Reply:
should be removed now unless i missed something